go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/hashicorp/terraform
 
+replace github.com/hashicorp/go-plugin v1.2.2 => ../go-plugin
```

Comment:
Please remove this before merging. :grinning:

Reply:
No. It's load bearing. :D

Reply:
(I was waiting for a tag to be cut; it's cut now, updating.)